src/main/java/org/opensearch/ubi/UbiActionFilter.java

```diff
@@ -303,4 +309,26 @@ private String getResourceFile(final String fileName) {
         }
     }
 
+    private void sendOtelTrace(final Task task, final Tracer tracer, final QueryRequest queryRequest) {
```

Comment:
@jzonthemtn  What exactly you are trying to capture here? The span is supposed to capture the duration of the particular operation which seems to be not the cases (start span when search kicks in, end when it is done).

 Are you trying to attach more attributes to the current span?

Reply:
The query (along with its associated UBI data) and creating an OTel span from it. I struggled with the appropriate OTel concept (trace vs log) for best describing a query.

When a query with UBI data comes in, it can either be 1) sent to Data Prepper `http` source, or 2) sent out as OTel to Data Prepper. 

An OTel log made more sense to me because it doesn't seem like a span since it's not capturing any operation, but a log didn't seem to fit structured data.

Reply:
> An OTel log made more sense to me because it doesn't seem like a span since it's not capturing any operation, but a log didn't seem to fit structured data.

:+1: